dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -25,7 +25,7 @@ static const char *ERROR_KIND = "Native";
 static const char *LOGGER_NAME = "crash";
 static const char *LOG_TAG = "DatadogNdkCrashReporter";
 static const char *SERVICE_NAME = "";
-static const char *STORAGE_DIR = nullptr;
+const char *STORAGE_DIR = nullptr;
```

Comment:
Why do you remove the static here ?

Reply:
cause I need to access externally and to modify it from the functional test NDK method. I will try to see if it actually works with static also.